holoviews/element/tiles.py

```diff
@@ -188,6 +191,8 @@ def wikimedia_replacement():
 OpenTopoMap = lambda: Tiles('https://a.tile.opentopomap.org/{Z}/{X}/{Y}.png', name="OpenTopoMap")
 Wikipedia = wikimedia_replacement
 
-tile_sources = {k: v for k, v in locals().items() if isinstance(v, FunctionType) and k not in
+_all_tile_sources = {k: v for k, v in locals().items() if isinstance(v, FunctionType) and k not in
                 ['ESRI', 'lon_lat_to_easting_northing', 'easting_northing_to_lon_lat',
                  'deprecation_warning', 'wikimedia_replacement']}
+
+tile_sources = {k: v for k, v in _all_tile_sources.items() if v.__name__ != 'deprecated_tilesource_warning'}
```

Comment:
Looks good to me!

As a sanity check I would grep the notebooks and source to see if `tile_sources` was used anywhere else where the deprecated tiles might have been expected (I don't think so). Otherwise, I'm happy with this approach.